index.html

```diff
@@ -282,14 +282,240 @@ <h2>Contact me</h2>
         <input type="text" title="First name" id="name" name="first_name" required placeholder="Your name"
           maxlength="30" />
         <input type="Email" title="email" id="email" name="email" required placeholder="username@example.com" />
-        <textarea title="Comments" id="comments" name="comments" required placeholder="Write your message here"
-          maxlength="500"></textarea>
-        <button type="submit" class="action-buttons">Get in touch</button>
+        <div class="errorMsgContainer">
+          <textarea title="Comments" id="comments" name="comments" required placeholder="Write your message here"
+            maxlength="500"></textarea>
+
+        </div>
+        <div>
+          <button type="submit" id="emailBtn" class="action-buttons">Get in touch</button>
+        </div>
       </form>
     </section>
   </div>
 
+  <!-- POP UP MENU BEGINS -->
+  <div class="modal" id="modal1">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h2>Tonic</h2>
+        <div class="modal-close"><img src="images/CancelIcon.svg" alt=""></div>
+      </div>
+      <ul class="popUpList">
+        <li>CANOPY</li>
+        <li>&#9679;</li>
+        <li>Back End Dev</li>
+        <li>&#9679;</li>
+        <li>2015</li>
+      </ul>
+      <div class="modal-content-placeholder">
+      </div>
+      <div class="popUpContent">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Deleniti, numquam
+          consequuntur? Assumenda quae explicabo quis. Enim et aliquam vitae voluptatem maiores itaque dignissimos
+          facilis deleniti, dolore, ipsa eveniet, nam nulla. Eos officiis maxime minus ratione tenetur ipsam, aut
+          fugit, consectetur voluptatibus sint totam in ex reprehenderit ab placeat ducimus laborum minima quo
+          doloremque
+          quaerat?<span class="popUpHide">At, corrupti eligendi! Quos ut culpa sed consequatur iusto? Et hic, placeat,
+            tempora labore quo
+            soluta eligendi, voluptates dolorum vero ad a inventore quos. Optio aut recusandae laudantium qui illum
+            facere atque blanditiis reiciendis molestias amet autem, eligendi assumenda. Eligendi incidunt et dicta,
+            dolore
+            aspernatur eveniet!</span></p>
+        <div class="footerContent">
+          <ul class="footerList">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            <li class="popUpHide">github</li>
```

Comment:
- [ ] Please remove all popup project cards elements from the `HTML` file it's supposed to be generated dynamically with javascript like the project requirements.

![image](https://user-images.githubusercontent.com/68796232/162477345-1e262ac6-87a1-4dae-a3cd-a1dd9870c6e5.png)

[Hint]: you can do it by storing the projects data like that:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
and then make a function to render these data on your page and make an event listener to the `see project` button to make this function work then make the styling for both mobile and desktop versions.